doc/Language/variables.pod6

```diff
@@ -265,12 +264,12 @@ X<|$?>
 
 Compile-time variables may be addressed via the C<?> twigil. They are known
 to the compiler and may not be modified after being compiled in. A popular
-example for this is:
+example is this:
```

Comment:
This was better before. Use the old version or bonus points for rewriting it to form a complete sentence.

You shouldn't say “A popular example is this.” but you could say “Here is a popular example of/for this.”). It should be noted I don't like that either… So either leave it as before or somehow make it more complete.